### PR TITLE
Fixing posgres datetime and timestamp column created with wrong format

### PR DESCRIPTION
**Pull request by zeotuan**
- Why:
See issue #4577 

- How: 
1. Change default useTz option of datetime and timezone to true in pg column compiler. 
2. Adding additional test to check when empty option object or option object wihout timezone is passed.

### CODE REVIEW

**Comment by kibertoad** [comment]
> Doesn't this change default values and hence is a breakibg change?

**Comment by kibertoad** [comment]
Isn't this still a breaking change, as it will invert values set by users already?

**Comment by zeotuan** [comment]
@kibertoad sorry the naming of variables was bad. Yeah i think so. I can change it so that it is not a breaking change anymore.

**Comment by kibertoad** [comment]
It is cleaner, yes, just a bit too late to change it, would require a semver major and still likely to break stuff for people who don't read docs.

**Comment by zeotuan** [comment]
Alright i Changed it. It shouldn't invert value already set by user anymore.

File: test/unit/schema-builder/postgres.js [hunk 1]
```diff
@@ -1168,6 +1181,19 @@ describe('PostgreSQL SchemaBuilder', function () {
     );
   });
 
+  it('adding datetime with timezone', () => {
+    tableSql = client
+      .schemaBuilder()
+      .table('users', (table) => {
+        table.datetime('foo', false);
```
Comment (armandAkop) [inline test/unit/schema-builder/postgres.js:1188]: Maybe I'm missing some context but wouldn't passing `false` here result in timestamp WITHOUT timezone? Why does it seem inverted?

Comment (zeotuan) [reply test/unit/schema-builder/postgres.js:1188]: - It's Because this is the original design 
image

- And this is the original test for timestamp() which do the same thing as datetime():
image

I did changed it to withTz so it does not seem inverted anymore. But as @kibertoad said, it will cause breaking change so i  just add the bug fix and sometest without changing the parameter. Maybe we could request this as feature for future update because i agree that it is not intuiitive. 
Also it is only mentioned in the documentation that you can use ```optiton: {useTz: boolean, precision: number}``` so i don't know if many people create timezone like that

**Comment by zeotuan** [comment]
@kibertoad I changed it so it fixed the bug without making breaking changes. Is there any other change that i need to do?

**Review by kibertoad — APPROVED** [review]

**Comment by kibertoad** [comment]
Thank you!
